staff/nicolas-martin/horroy/scripts/horroy.test.js

```diff
@@ -0,0 +1 @@
+// TODO do it nice!
```

Comment:
You don't need that file, right?

Reply:
I think in this project we don't need testing.

Reply:
the test is inside the horroy.spec.js file. Now, I have understood